### PR TITLE
some eli5 comments within the Dockerfile

### PR DESCRIPTION
**Pull request by shadeofblue**

### CODE REVIEW

**Comment by shadeofblue** [comment]
closes https://github.com/golemfactory/yagna-docs/issues/52

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,18 @@
 # Dockerfile used to build the VM image which will be downloaded by providers.
 # The file must specify a workdir and at least one volume.
+
+# We're using python slim image in this example to limit the time it takes for the
+# resultant image to be first downloaded by providers, given the fact that our example
+# here is limited to barebones Python installation.
+#
+# In a production app, one might wish to utilize a more full-featured Python image.
```
Comment (kmazurek) [inline Dockerfile]: Actually, I think trying to use minimal images in production apps can be considered a good practice (for the sake of the final image size). Therefore, I'd skip this sentence from the comment.

**Review by kmazurek — APPROVED** [review]
One tiny remark, looks good otherwise.

File: Dockerfile (hunk 1 above)
Comment (shadeofblue) [inline Dockerfile]:
```suggestion
```
